src/Secure/OpenIdConnectAuthentication.php

```diff
@@ -95,8 +105,11 @@ protected function login(): void
     protected function validateToken(): array
     {
         $validationUrl    = "https://login.twinfield.com/auth/authentication/connect/accesstokenvalidation?token=";
+        $validationResult = false;
```

Comment:
It seems that this can be simplified somewhat.

If there is no Office, the validationResult will always be an `OAuthException` right? Can you update the code here to reflect that?

Reply:
I'm pretty sure the `Office` has no relation to the process of fetching the access token (OAuth2 flow) nor validating it (the url only takes the access token as argument, and no headers).

So, you can have a valid token without supplying an Office instance.